### PR TITLE
Replace .format() with f-strings

### LINKED ISSUE #506: Replace .format() with f-strings
Used in some places but not everywhere yet

### PR DESCRIPTION
**Pull request by A1fus**
Closes #506

### CODE REVIEW

File: airsenal/framework/api_utils.py [hunk 1]
```diff
@@ -292,7 +290,7 @@ def best_transfer_suggestions(n_transfer, session_id, dbsession=DBSESSION):
     for p in players:
         added_ok = t.add_player(p)
         if not added_ok:
-            raise RuntimeError("Cannot add player {}".format(p))
+            raise RuntimeError("fCannot add player {p}")
```
Comment (jack89roberts) [inline airsenal/framework/api_utils.py:293]:
```suggestion
            raise RuntimeError(f"Cannot add player {p}")
```

**Comment by nbarlowATI** [comment]
@A1fus thanks for your help with this!! :)   Do you think this PR is ready to merge now, or is there more that you're looking to do?

**Comment by A1fus** [comment]
@nbarlowATI Happy to finish the job and convert the remaining ones, which branch should I point at now `develop` is deleted?

**Comment by jack89roberts** [comment]
I deleted it by mistake after we merged some changes yesterday, `develop` is back now 🙂

**Comment by A1fus** [comment]
Okay think that's everywhere it makes sense, ready for a review here now

**Review by jack89roberts — APPROVED** [review]
Thanks @A1fus , there were a lot more `.format` calls left than I realised!

**Pull request merged**